Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.9-slim
```

Comment:
The test suite is ran on python 3.10.7, is that an option to configure here?

I expect it all to be compatible but for consistency.

Reply:
Yeah, I see a `3.10.7-slim` tag for `python`, so that is possible. I just copied this Dockerfile from the Streamlit website and modified the important parts. I can change the base image too.